examples/ethereum-tracker/src/contract.rs

```diff
@@ -30,6 +30,7 @@ impl Contract for EthereumTrackerContract {
     type Message = ();
     type InstantiationArgument = InstantiationArgument;
     type Parameters = ();
+    type EventValue = ();
```

Comment:
Maybe just `Event`?

Reply:
We already have an `Event` type which includes the event ID. I wanted to avoid confusion with that. Not sure.